decision_graph/integration.py

```
@@ -0,0 +1,712 @@
+                        loop = asyncio.get_event_loop()
+                        if loop.is_running():
+                            # Ensure worker is started and enqueue
+                            async def enqueue_job():
+                                await self.ensure_worker_started()
+                                await self.worker.enqueue(
+                                    decision_id=decision_id,
+                                    priority="low",
+                                    delay_seconds=5,
+                                )
+
+                            asyncio.create_task(enqueue_job())
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)